apis/kueue/v1beta1/workload_types.go

```diff
@@ -124,6 +129,22 @@ type WorkloadStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// reclaimablePods keeps track of the number pods within a podset for which
+	// the resource reservation is no longer needed.
+	// +optional
+	// +listType=map
+	// +listMapKey=name
+	ReclaimablePods []ReclaimablePod `json:"reclaimablePods,omitempty"`
```

Comment:
I'm having second thoughts here.

If we assume that the admission section can be modified after admission (to update Count and usage), then I prefer we have `reclaimableCount` inside the admission struct.

I think this might be a better idea as we move towards elastic jobs. @tenzen-y ?

Reply:
> then I prefer we have reclaimableCount inside the admission struct.

Does this mean we have `reclaimableCount` inside the admission instead of having this `ReclaimablePods` (removing `ReclaimablePods `)?

Reply:
yes, that's what I'm thinking.

Reply:
IIRC, we selected `ReclaimablePods` instead of `reclaimableCount` since it could be misleading when the job is suspended.

https://github.com/kubernetes-sigs/kueue/pull/742#discussion_r1187191726

However, as @alculquicondor says, we can ignore that concern if we have `reclaimableCount` inside the `Admission`.

> I think this might be a better idea as we move towards elastic jobs.

right. The `Admission` has the `reclaimableCount` would be more natural for the elastic job.

@trasc Are you concerned about having `reclaimableCount` inside the `Admission`?

Reply:
just to look similar to flavors and usage